test/format/ops.c

```diff
@@ -318,7 +318,7 @@ operations(u_int ops_seconds, u_int run_current, u_int run_total)
              */
             thread_ops = -1;
             ops_seconds = 0;
-            g.stop_timestamp = (GV(RUNS_OPS) * run_current) / run_total;
+            g.stop_timestamp = (uint64_t) ((GV(RUNS_OPS) * run_current) / run_total);
```

Comment:
My reading of the Coverity warning indicates to me that this will not address the issue, but I am curious if it will suppress the message. I recommend that yourself, Sean and meet to discuss.

Reply:
fixed https://github.com/wiredtiger/wiredtiger/pull/9982/commits/212092b51b8b74c393ab3b2afc8436d757e83cef